content/en/docs/tasks/administer-cluster/dns-debugging-resolution.md

```diff
@@ -95,7 +95,7 @@ Use the `kubectl get pods` command to verify that the DNS pod is running.
 
 For CoreDNS:
 ```shell
-kubectl get pods --namespace=kube-system -l k8s-app=coredns
+kubectl get pods --namespace=kube-system -l k8s-app=kube-dns
```

Comment:
@mhamdisemah This seems like the wrong fix. [Line 96](https://github.com/kubernetes/website/pull/14816/files#diff-108b8d55763a7f2d881b894d15e12901R96) labels this as for CoreDNS, and I think that's correct.

Reply:
I think the fix needs to happen on [line 163](https://github.com/kubernetes/website/pull/14816/files#diff-108b8d55763a7f2d881b894d15e12901R163) onwards.

Reply:
what is the problem for line 163 it is kube-dns

Reply:
the problem as described in the bug it is 

> Problem:
> kubectl get pods --namespace=kube-system -l k8s-app=coredns 
> command does not give the output as it is seen in the documentation because coredns pod has still the label " k8s-app=kube-dns"

Reply:
Oops I think you were right the first time @mhamdisemah 

I'm not sure what I was thinking, but it seems I wasn't with it.